app/src/components/MiniGlobe.jsx

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import MiniGlobeStyles from 'styles/components/miniGlobe.scss';
+import { geoOrthographic, geoPath } from 'd3-geo'; // eslint-disable-line
+import { feature } from 'topojson-client';
+import { MINI_GLOBE_SETTINGS } from 'config';
+
+const jsonData = require('assets/topoJson/ne_110m_land.json');
+
+class MiniGlobe extends Component {
+
+  constructor(props) {
+    super();
+    this.worldData = feature(jsonData, jsonData.objects.land).features;
+    const viewportWidth = props.viewportWidth;
+    const viewportHeight = props.viewportHeight;
+    const size = this.markerSize(props.zoom, viewportWidth, viewportHeight);
+
+    this.setState({
+    });
+    this.state = {
+      projection: null,
+      markerWidth: size.width,
+      markerHeight: size.height
+    };
+  }
+
+  markerSize(zoom, viewportWidth, viewportHeight) {
```

Comment:
this method is called twice, and each time after it's used you set the result to the state. Perhaps it should be named "setMarkerSize" and it should also update the state.